baleful/anal/anal_baleful.c

```diff
@@ -598,8 +623,12 @@ static int set_reg_profile(RAnal *anal) {
 	return r_reg_set_profile_string (anal->reg, p);
 }
 static int esil_baleful_init (RAnalEsil *esil) {
-	if (!esil)
+	if (!esil) {
 		return false;
+	}
+	bool ret = r_anal_esil_set_interrupt (esil, r_anal_esil_interrupt_new (esil, 0,  &bale_intr_handler00));
+	ret &= r_anal_esil_set_interrupt (esil, r_anal_esil_interrupt_new (esil, 0,  &bale_intr_handler04));
```

Comment:
why keep setting interrupts if one fails? also maybe it should unset if the hook fails